app.py

```diff
@@ -28,8 +28,8 @@
           "allows you to compare the main metrics between two releases."))
 
 # Load data
-files = glob.glob("data/*.csv")
-dfs = [pd.read_csv(f) for f in files]
+files = glob.glob("data2/*.csv")
```

Comment:
Is that an intended change? It looks like all of the files are still in `data/`

Reply:
That was a typo! Thank you for pointing it out.
Fixed in f292bba.